components/Ecosystem/EcosystemItemCard.vue

```diff
@@ -4,10 +4,9 @@
       class="ecosystem-item-card"
       :title="member.name"
       :tags="member.labels"
-      :tooltip-tags="[
+      :primary-tags="[
         {
           label: member.tier,
-          description: tierDescription,
         },
       ]"
```

Comment:
We will only have one primary tag, so I think the following implementation approaches better what we want:
```
:primary-tag="member.tier"
```

Reply:
updated in https://github.com/Qiskit/qiskit.org/pull/3322/commits/033418cf80dca3b0a41e7aaeccb0cc63685ed2bc